src/libsync/syncfileitem.cpp

```diff
@@ -73,7 +73,7 @@ SyncFileItemPtr SyncFileItem::fromSyncJournalFileRecord(const SyncJournalFileRec
     item->_remotePerm = rec._remotePerm;
     item->_serverHasIgnoredFiles = rec._serverHasIgnoredFiles;
     item->_checksumHeader = rec._checksumHeader;
-    item->_encryptedFileName = QString::fromUtf8(rec._e2eMangledName);
+    item->_encryptedFileName = rec.e2eMangledNameFromUtf8();
```

Comment:
I would be concerned about testing this enough...

Reply:
> I would be concerned about testing this enough...

@camilasan I'd say this one is pretty straightforward. QString::fromUtf8(rec._e2eMangledName); is just called from the outside :)

Reply:
I meant where fromSyncJournalFileRecord is being used.